appengine/flexible/numpy/main.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START app]
+import logging
+
+from flask import Flask
+import numpy as np
+
+app = Flask(__name__)
+
+
+@app.route('/calculate')
+def calculate():
+    return_str = ''
+    x = np.array([[1, 2], [3, 4]])
+    y = np.array([[5, 6], [7, 8]])
+
+    return_str += 'x: {} , y: {}<br />'.format(str(x), str(y))
+
+    # Multiply matrices
+    return_str += 'x dot y : {}'.format(str(np.dot(x, y)))
+    return return_str
+
+
+@app.route('/')
```

Comment:
sgtm but I feel like this wasn't done in DPE bot so wasn't sure what our standard was (I know that's different repo)